### ISSUE TITLE
Server error using cursor pagination

### ISSUE THREAD

**Issue by gepd** [appwrite/appwrite#3403]
### 👟 Reproduction steps

Using the following code returns a server error 

```js
listDocuments("collection", undefined, 6, undefined, cursor, "after", ["$id"], ["DESC"]);
```

### 👍 Expected behavior

Return a list of documents 

### 👎 Actual Behavior

`{"message":"Server Error","code":500,"type":"general_unknown","version":"0.14.2"}`

docker logs appwrite

```
[Error] Method: GET
[Error] URL: /v1/database/collections/:collectionId/documents
[Error] Type: Appwrite\Extend\Exception
[Error] Message: Order attribute not found in schema: _uid
[Error] File: /usr/src/code/app/controllers/api/database.php
[Error] Line: 1778
[Error] Timestamp: 2022-06-20T04:51:45+00:00
```

### 🎲 Appwrite version

Version 0.14.x - 0.15.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by gepd** [comment]
I got the same result using the new `$updatedAt` or `$createdAt`

```
[Error] Message: Order attribute '_updatedAt' is empty.
[Error] File: /usr/src/code/vendor/utopia-php/database/src/Database/Adapter/MariaDB.php
[Error] Line: 900
```
there any workaround for this?

**Comment by Meldiron** [comment]
I just realized the PR did not directly address issues with dates. Will investigate those further.

**Comment by gepd** [comment]
I'll close this as this was solved in `0.15.3`
